packages/vscode-apollo/src/testRunner/jest-vscode-environment.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Exposes the Visual Studio Code extension API to the Jest testing environment.
+ * For custom environments reference:
+ * @see https://jestjs.io/docs/en/configuration.html#testenvironment-string
+ */
+
+// Using import here results in `jest_environment_node_1.default`
+// The `.default` in this case doesn't exist. Interop issue with the built
+// version of the package?
+const NodeEnvironment = require("jest-environment-node");
+import * as vscode from "vscode";
+
+class VsCodeEnvironment extends NodeEnvironment {
+  constructor(config: any) {
+    super(config);
+  }
+
+  public async setup() {
+    await super.setup();
+    this.global.vscode = vscode;
+
+    // For some reason this seems to be required for the Jest output to be streamed
+    // to the Debug Console.
+    process.stdout.write = this.logger;
```

Comment:
yikes haha. Not sure of the impact here, but we should probably reset these values in the teardown?

Reply:
👍 I'd be interested in exploring this with you at some point, maybe there's a better way to go about this.